.idea/codeStyles/Project.xml

```diff
@@ -1,5 +1,23 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
```

Comment:
Nice work @epicadk :tada: but these additions aren't needed. You can remove `./idea` files by adding them to `.gitignore`